__tests__/Treebeard.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
```

Comment:
why you don't use a `<component>.test.js` or `<component>.specs.js` in the filename?